src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/PaintCan.java

```diff
@@ -449,9 +450,8 @@ else if (dye1 == DyeColor.PINK && dye2 == DyeColor.PURPLE) {
         else if (dye1 == DyeColor.WHITE && dye2 == DyeColor.GRAY) {
             return DyeColor.LIGHT_GRAY;
         }
-        else {
-            // colors don't mix
-            return null;
-        }
+
+        // colors don't mix
+        return null;
```

Comment:
Why did you remove the `else` here?

Reply:
Makes Sonar happy :lul: All it does is reduce cognitive complexity by 1 bringing the total count to the 20 Sonar likes, it's nothing exceptionally useful really. I liked it more with the else as it seems clearer to me but this isn't that bad and reduces technical debt so I didn't see a good reason not to

Reply:
:NotLikeThis:

Reply:
While I'm adding the else for the comment above, do you want me to revert this change or?

Reply:
yeah sure